manual/src/cmds/ocamldep.etex

```diff
@@ -3,18 +3,20 @@
 
 The "ocamldep" command scans a set of OCaml source files
 (".ml" and ".mli" files) for references to external compilation units,
-and outputs dependency lines in a format suitable for the "make"
-utility. This ensures that "make" will compile the source files in the
-correct order, and recompile those files that need to when a source
-file is modified.
+and outputs dependency lines in a subset of "make" syntax understood by
+make and other build tools, such as \href{https://dune.build/}{dune},
+\href{https://ninja-build.org/manual.html#ref_headers}{ninja}, and
+\href{https://buck2.build/docs/rule_authors/dynamic_dependencies/}{buck2}.
+The dependency information is used by build tools compile source fles in the correct order,
+and recompile those files that need to when a source file is modified.
 
 The typical usage is:
 \begin{alltt}
         ocamldep \var{options} *.mli *.ml > .depend
 \end{alltt}
 where "*.mli *.ml" expands to all source files in the current
 directory and ".depend" is the file that should contain the
-dependencies. (See below for a typical "Makefile".)
+dependencies. (See below for an example "Makefile".)
```

Comment:
I am not sure why you are replacing `typical` by `example`?

Reply:
Two ways in which the makefile is not typical:
- The makefile doesn't really make sense: `ocamldep` is used to discover dependency structure, but then dependency instruction must be hard-coded for linking
- It's my understanding that it's not typical to build OCaml with makefiles


From the issue discussion:

- https://github.com/ocaml/ocaml/issues/12925#issuecomment-1902636228
- https://github.com/ocaml/ocaml/issues/12925#issuecomment-1902624139

Reply:
This is a typical makefile for an OCaml project using `make`.